course-definition.yml

```diff
@@ -159,8 +170,6 @@ stages:
         "go-tcp-server",
         "rust-tcp-server",
         "python-tcp-server",
-        "java-tcp-server",
```

Comment:
@UdeshyaDhungana let's rebase on main - looks like you unintentionally removed a few of these